Gemfile

```diff
@@ -3,6 +3,7 @@ source "https://rubygems.org"
 gem "cocoapods"
 gem "cyclonedx-cocoapods"
 gem "fastlane"
+gem "xcodeproj"
```

Comment:
We only need this to drive the `diff-xcodeproj.sh` script, but since it's already a dependency of CocoaPods then it's not adding anything to depend on it directly, and that script feels like it'll come in handy.